planning/behavior_path_planner/src/scene_module/avoidance/avoidance_module.cpp

```diff
@@ -294,11 +294,19 @@ ObjectDataArray AvoidanceModule::calcAvoidanceTargetObjects(
 
   // debug
   {
+    debug_data_.avoidance_debug_msg_array.avoidance_info.clear();
     auto & debug_data_avoidance = debug_data_.avoidance_debug_msg_array.avoidance_info;
     debug_data_avoidance = avoidance_debug_msg_array;
-    debug_data_avoidance.insert(
-      debug_data_avoidance.end(), debug_avoidance_initializer_for_shift_point_.begin(),
-      debug_avoidance_initializer_for_shift_point_.end());
+    if (!debug_avoidance_initializer_for_shift_point_.empty()) {
+      const bool is_info_old_ =
+        (clock_->now() - debug_avoidance_initializer_for_shift_point_time_).seconds() > 1.0;
```

Comment:
Everything is great, but just to confirm, is there any reason why @tkimura4-san choose 1.0 second and not faster time?

Reply:
This parameter is not well-reasoned.
 I think that maybe 0.1 ~ 0.2 seconds is also okay.

Reply:
I see. Yes, I think 0.1 second should be great.

Also, is it possible to convert this part into function? I think it might help to improve readability.

Reply:
Thank you for your comment.
I fixed at [32eace2](https://github.com/autowarefoundation/autoware.universe/pull/1068/commits/32eace2b4b7c9be96d3f7b04e7c6ba6b836ab42b)